content/blog/lenka-segura-interview.md

```diff
@@ -0,0 +1,74 @@
+
+**Tell us more about your current job. You are working on cancer research. Was it your goal to get to this field or more of a coincidence? And what exactly is your daily work?**
+
+It was very difficult to get the confidence to start applying for jobs. I was staring at the job offers coming through the Pyladies and Codebar mailing lists. At this stage, the selection process was a bit peculiar, I would look at the offers and quickly find out reasons I should not apply. Once my eyes stopped into this job position in the cancer research computational lab… I just couldn’t find any excuse not to apply. It spoke to my research background, it was really meaningful and it was an entry level position. The interview went well and the team was very welcoming.
+
+The project I work on is to build an application which gathers known and also computed information about mutations found in cancer genomes. My daily work has been evolving during my tenure in the lab. At first it was just taking ownership of some legacy projects and coding a replacement, then I started integrating what I did with the existing stack. A bit later I started gathering requirements and improving the UX and lately I’ve started handling some of the operation analysis web apps we have. A little bit of everything.
+
+**As I mentioned at the beginning of our interview, I saw you speaking at DevConf CZ. If I remember correctly, it was your first time as a speaker. Why did you decide to talk at DevConf? How was the experience?**
+
+I had previously attended DevConfCZ thanks to the Outreachy travel stipend and thoroughly enjoyed the experience. Having been a previous attendee I received the call for papers email and I started to think about how conferences are from the speaker side. Fortunately Devconf offers a Speaker Guide and an assistance of a Speaker Coach for new speakers. The Speaker Coach contacted me some time before the conference and helped me to improve the slides and practice the talk.
+
+**One of the long-term goals of the CyberMagnolia community is to put together a group of female tech speakers. We want to increase their visibility and make it easier for the conference organizers to reach them. Would you consider speaking at some more conferences when the situation allows?**
+
+Absolutely!
+
+**You took quite a journey from agricultural chemistry to cancer research. What would you say has been the biggest challenge in your career so far?**
```

Comment:
It feels like this is a direct follow up to the question you asked in line 43. Do you think it would make sense to move this question and answer up, to go before DevConf question?

Reply:
Good point, thanks!